src/xdocs/config_blocks.xml

```diff
@@ -110,6 +85,63 @@ switch (a)
         <source>
 &lt;module name=&quot;AvoidNestedBlocks&quot;/&gt;
         </source>
+
+        <p> Example: </p>
```

Comment:
no need for extra leading/trailing spaces

Reply:
Done